src/bentoml/_internal/cloud/deployment.py

```diff
@@ -45,13 +46,7 @@ def for_hpa_conf(cls, **kwargs) -> DeploymentTargetHPAConf:
 
     @classmethod
     def for_runner(cls, **kwargs) -> DeploymentTargetRunnerConfig:
-        exclusive_api_server_key = {
-            v for v in kwargs if v not in attr.fields_dict(DeploymentTargetRunnerConfig)
-        }
-        return bentoml_cattr.structure(
-            {k: v for k, v in kwargs.items() if k not in exclusive_api_server_key},
-            DeploymentTargetRunnerConfig,
-        )
```

Comment:
Why are we removing this?

Reply:
I guess we set forbid_extra_keys to False
we should just set forbid_extra_keys to true